assets/js/directives/student_sidebar_directive.js

```diff
@@ -1,9 +1,9 @@
 app.directive("studentSidebarDirective", ["$route", "Module", function($route, Module) {
   return {
-    templateUrl: "/js/templates/student_sidebar_template.html",
-    link: function(scope, elem, attr) {
-      Module.query({library: $route.current.params.libraryId}, function(modules) {
-        scope.library = modules;
+    templateUrl: "/js/templates/directives/student_sidebar_template.html",
+    link: function(scope, attr, elem) {
+      Module.query({library: $route.current.params.libraryId}, function(data) {
+        scope.library = data;
```

Comment:
This seems to be named improperly. You're not talking about a library, you're talking about an array of modules. This could confuse other engineers and burn their time.